distributed/worker.py

```diff
@@ -2292,6 +2307,48 @@ def put_key_in_memory(self, ts, value, transition=True):
 
         self.log.append((ts.key, "put-in-memory"))
 
+    def handle_annotators(
+        self,
+        ts: TaskState,
+        value: object = no_value,
+    ):
+        """Annotate task if applicable and report back to the scheduler"""
+
+        # Separate annotation updates into two cases:
+        a1 = {}  # when the task's dependents should also be updated
+        a2 = {}  # when only the task itself should be updated
```

Comment:
Do you have thoughts on better names here?

Reply:
Heh, yeah I can do that :)